### PR TITLE
Done. Leaflet css cannot be removed because someone edited it. Please…

### PR DESCRIPTION
**Pull request by milovanderlinden**
Please review this one. It took me a lot more time then estimated because preact does not play nice with mapbox ;-) but it is fixed and now acts the same as the leaflet variant.

Please check all code and if someone can undo the changes done to leaflet.css and deliver the changes to the master style file, that would be good.

### CODE REVIEW

File: src/components/WorldMap.js [hunk 1]
```diff
@@ -118,6 +126,7 @@ export class WorldMap extends Component {
   render() {
     return html`
       <div
+        class="leaflet-container"
```
Comment (milovanderlinden) [inline src/components/WorldMap.js]: this one needs fixing

**Comment by LarsDenBakker** [comment]
The issue with loading mapbox-gl is that it isn't a native es module, so it needs to some build transformation step to make it work in the browser as a module. I found an open issue on this https://github.com/mapbox/mapbox-gl-js/issues/8676

We could add https://github.com/rollup/plugins/tree/master/packages/commonjs to rollup to fix this, but then we will need to run it during development as well. If it's one dependency, putting it on the window is fine for now.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,14 @@
 {
   "name": "lockdown",
   "version": "0.0.0",
+  "description": "A reactp web application for displaying the lockdown status of various countries",
```
Comment (thepassle) [inline package.json]:
```suggestion
  "description": "A preact web application for displaying the lockdown status of various countries",
```

File: src/components/WorldMap.js (hunk 1 above)
Comment (thepassle) [reply src/components/WorldMap.js]: fixed

**Comment by LarsDenBakker** [comment]
I know we had some discussion on this yesterday, sorry to bring it up again but mapbox-gl is larger than leaflet: https://bundlephobia.com/result?p=mapbox-gl@1.9.0

It can be perfectly fine as a choice, if the features and functionalities are required. Just want to make sure we make a conscious decision here.

It depends on the requirements of the project. If we expect the app to be used on low-end mobile phones we do need to watch the filesize.

**Comment by thepassle** [comment]
This is mostly my bad, when I checked bundlephobia I was looking at: https://bundlephobia.com/result?p=mapbox@1.0.0-beta10, which is a lot smaller than leaflet, and also not the correct package.

Considering that mapbox is 184.4kB vs leaflets 40.1kB, I think we should stay with leaflet

@milovanderlinden Sorry for causing the confusion 🙁

**Comment by thepassle** [comment]
Closing, we decided to go with leaflet for now

**Pull request closed without merging**